<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassportDialog</class>
 <widget class="QDialog" name="PassportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Основное окно */
QWidget {
    background-color: #f4f6f9;   /* светлый фон */
    font-family: Segoe UI, Arial, sans-serif;
    font-size: 12pt;
    color: #2c3e50;
}

/* Заголовки */
QLabel {
    font-weight: bold;
    font-size: 13pt;
    color: #1a1a1a;
}

/* Поля ввода */
QLineEdit, QDateEdit, QComboBox, QTextEdit {
    background-color: #ffffff;
    border: 1px solid #b0bec5;
    border-radius: 6px;
    padding: 4px 6px;
}

QLineEdit:focus, QDateEdit:focus, QComboBox:focus, QTextEdit:focus {
    border: 1px solid #1976d2;
    background-color: #f9fcff;
}

/* Кнопки */
QPushButton {
    background-color: #1976d2;
    color: #ffffff;
    border-radius: 6px;
    padding: 6px 12px;
    font-weight: 600;
}

QPushButton:hover {
    background-color: #1565c0;
}

QPushButton:pressed {
    background-color: #0d47a1;
}

/* Секции */
QGroupBox {
    border: 1px solid #b0bec5;
    border-radius: 8px;
    margin-top: 10px;
    padding: 8px;
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 5px;
}

/* Скроллбар */
QScrollBar:vertical {
    border: none;
    background: #e0e0e0;
    width: 10px;
    margin: 0px;
    border-radius: 5px;
}

QScrollBar::handle:vertical {
    background: #90a4ae;
    min-height: 20px;
    border-radius: 5px;
}

QScrollBar::handle:vertical:hover {
    background: #607d8b;
}
</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>671</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="10" column="0">
     <widget class="QComboBox" name="comboBoxGender"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEditName"/>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEditKemVidan"/>
    </item>
    <item row="10" column="1" colspan="2">
     <widget class="QComboBox" name="comboBoxGrazdanstvo"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditMiddleName"/>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLineEdit" name="lineEditIDPassport">
      <property name="placeholderText">
       <string>Уникальный ID</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLineEdit" name="lineEditBirthPlace"/>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEditSeria"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEditLastName"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditNumberPassport"/>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="lineEditCodePodr"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEditAddress"/>
    </item>
    <item row="13" column="2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Дата Выдачи</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>День рождения</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1" colspan="2">
     <widget class="QDateEdit" name="dateEditBirthDay"/>
    </item>
    <item row="11" column="1" colspan="2">
     <widget class="QDateEdit" name="dateEditDateVidachi"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PassportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PassportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
